docs/sources/helm-charts/mimir-distributed/get-started-helm-charts/_index.md

```diff
@@ -159,6 +159,20 @@ In the example that follows, metamonitoring scrapes metrics about Grafana Mimir
 
    > **Note:** In a production environment the `url` above would point to an external system, independent of your Grafana Mimir instance, such as an instance of Grafana Cloud Metrics.
 
+1. Download the Grafana Agent Operator Custom Resource Definitions (CRDs) required to enable metamonitoring from
+   https://github.com/grafana/agent/tree/main/production/operator/crds
+
+   Install the CRDs on your cluster:
+
+   ```bash
+   kubectl apply -f production/operator/crds/
+   ```
+
+   Helm will only install Custom Resource Definitions on an initial chart install,
+   [not on a chart upgrade](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/).
+   If metamonitoring was not enabled when the chart was first installed,
+   the CRDs must be installed manually before a Helm upgrade to enable metamonitoring.
+
```

Comment:
```suggestion
1. Enable metamonitoring:
    1. Download the Grafana Agent Operator Custom Resource Definitions (CRDs) from https://github.com/grafana/agent/tree/main/production/operator/crds.

   1. Install the CRDs on your cluster:

   ```bash
   kubectl apply -f production/operator/crds/
   ```
   Helm only installs Custom Resource Definitions on an initial chart installation, and not on a chart upgrade. For details, see [Some caveats (and explanations)](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations).
   If you did not enable metamonitoring when the chart was first installed,
   you must manually install the CRDs before performing a Helm upgrade to enable metamonitoring.

```

Reply:
These indentations need to be test.

Reply:
The suggestions regarding the list ordering don't make as much sense in the full context of the page.

The section is called `Generate some metrics for testing` and the intro paragraph has two references to the fact that the whole section is describing how to enable metamonitoring, in order to generate metrics for testing.

Step 2 above this new step reads `To enable metamonitoring... [add some YAML snippet]`

The suggestion here is to make step 3 titled `Enable metamonitoring:`, then have sub-steps under that. This is choppy and does not make sense, as every top-level step in the list is a step for enabling metamonitoring.

What we could do is swap the step order to better mirror the instructions here, which are the clearest we have:
https://grafana.com/docs/helm-charts/mimir-distributed/latest/run-production-environment-with-helm/#monitor-the-health-of-your-grafana-mimir-cluster

So the steps go:

1. Download the CRDs
2. Apply the CRDs
3. Add the custom YAML
4. Apply the custom YAML (in the above link this is implied by 3, but not explicitly said)

Reply:
Go with whatever order is technically the most accurate.